openpype/modules/sync_server/launch_hooks/pre_copy_last_published_workfile.py

```diff
@@ -111,45 +112,15 @@ def execute(self):
         asset_doc = self.data.get("asset_doc")
         anatomy = self.data.get("anatomy")
 
-        context_filters = {
-            "asset": asset_name,
-            "family": "workfile",
-            "task": {"name": task_name, "type": task_type}
-        }
-
-        # Add version filter
-        workfile_version = self.launch_context.data.get("workfile_version", -1)
-        if workfile_version > 0 and workfile_version not in {None, "last"}:
-            context_filters["version"] = self.launch_context.data[
-                "workfile_version"
-            ]
-
-            # Only one version will be matched
-            version_index = 0
-        else:
-            version_index = workfile_version
-
-        workfile_representations = list(get_representations(
+        workfile_representation = get_last_workfile_representation(
             project_name,
-            context_filters=context_filters
-        ))
-
-        if not workfile_representations:
-            self.log.debug(
-                'No published workfile for task "{}" and host "{}".'.format(
-                    task_name, host_name
-                )
-            )
-            return
-
-        filtered_repres = filter(
-            lambda r: r["context"].get("version") is not None,
-            workfile_representations
+            asset_name,
+            task_name,
+            asset_doc=asset_doc,
         )
-        # Get workfile version
-        workfile_representation = sorted(
-            filtered_repres, key=lambda r: r["context"]["version"]
-        )[version_index]
+        if not workfile_representation:
+            self.log.debug("No workfile representation found.")
```

Comment:
Can we get back to the more informational log that was there before?
```suggestion
            self.log.debug(
                'No published workfile for task "{}" and host "{}".'.format(
                    task_name, host_name
                )   
            )
```